vclib/src/commonTest/kotlin/at/asitplus/wallet/lib/agent/AgentSdJwtTest.kt

```diff
@@ -38,10 +45,29 @@ class AgentSdJwtTest : FreeSpec({
 
     "simple walk-through success" {
         issueDummyCredentials(holder, issuer, holderCryptoService)
-        val vp =
-            holder.createPresentation(challenge, verifier.identifier, requestedClaims = listOf("given-name")).also {
-                it.shouldNotBeNull()
-            }
+        val presentationParameters =
+            holder.createPresentation(
+                challenge,
+                verifier.identifier,
```

Comment:
Code here seems duplicated with functions below

Reply:
fixed in commit 950b306a68c5b7a9af3591d61dead5fe364bd94c